### PR TITLE
Feat/Bonus Payable 

### PR DESCRIPTION
**Pull request by ishimwe-rodrigue**
Task description
view items in the bonus payable page
Disable previous dates on the begin date

### CODE REVIEW

File: app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        this.initialOdata = new InitializationHelper(this.getI18nText.bind(this));
+        this.getBonusPayable();
+    }
+
+    public async getBonusPayable(): Promise<void> {
+        const oModel = this.getModel("trancheModel") as ODataModel;
+
+        try {
+            // Fetch data using OData read
+            const oContext = await oModel.bindList("/bonusPayable").requestContexts(0, 100);
```
Comment (Michael-amali) [inline app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts:28]: Explain what this line of code does, .requestContexts(0, 100);

File: app/amalisov.cuibono/webapp/i18n/i18n.properties [hunk 2]
```diff
@@ -117,6 +117,7 @@ deleteError= Failed to delete tranche
 FailedOverride=Failed to override amounts
 errorUnexpected=Unexpected error occurred
 erroLoading=Error loading fragment
+errorDispayingData= erro dispaying Bonus Payable data
```
Comment (Christopher-Angoe) [inline app/amalisov.cuibono/webapp/i18n/i18n.properties:120]: missing an r at the end of error

**Review by Michael-amali — CHANGES_REQUESTED** [review]
Check comment

File: app/amalisov.cuibono/webapp/i18n/i18n.properties (hunk 2 above)
Comment (Michael-amali) [reply app/amalisov.cuibono/webapp/i18n/i18n.properties:120]: And the "displaying"

File: app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts (hunk 1 above)
Comment (ishimwe-rodrigue) [reply app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts:28]: this line is used the fetch the request 100 items so 0, 100  are the index

Comment (Michael-amali) [reply app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts:28]: So if there are more than 100 items, what happens ?

Comment (ishimwe-rodrigue) [reply app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts:28]: it doesnt show more than 100 as the method requestContexts show the length of 100 items even though i removed (0, 100) as i think 100 is the default

**Review by Christopher-Angoe — APPROVED** [review]

File: app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts (hunk 1 above)
Comment (Michael-amali) [reply app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts]: What really ?

Comment (ishimwe-rodrigue) [reply app/amalisov.cuibono/webapp/controller/BonusPayable.controller.ts]: i tried it doesnt work i even asked for help from team c i decided to use the controller
maybe i can show it you

**Review by Michael-amali — APPROVED** [review]
ok

**Pull request merged**